modules/material/material.service.js

```diff
@@ -12,17 +12,18 @@ const materialErrorMessage = JSON.stringify([
 ]);
 class MaterialsService {
   async getAllMaterials() {
-    return (await Material.find()) || new Error(materialErrorMessage);
+    const material = await Material.find();
+    return material;
   }
 
   async getMaterialById(id) {
-    return (await Material.findById(id)) || new Error(materialErrorMessage);
+    return (await Material.findById(id)) || new Error(MATERIAL_NOT_FOUND);
   }
 
   async updateMaterial(id, material) {
     return (
       (await Material.findByIdAndUpdate(id, material))
```

Comment:
Please, add third parameter ` {new: true}` to return already updated entity

Reply:
fixed